jetty-http2-agent/src/main/java/com/palantir/conjure/java/http2/Http2Agent.java

```diff
@@ -16,41 +16,33 @@
 
 package com.palantir.conjure.java.http2;
 
-import com.ea.agentloader.AgentLoader;
 import java.lang.instrument.Instrumentation;
-import org.mortbay.jetty.alpn.agent.Premain;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-/** A very simple wrapper around the jetty-alpn-agent for dynamic loading. */
+/**
+ * No-op.
+ * @deprecated this class no longer has any effect.
+ */
+@Deprecated
 public final class Http2Agent {
     private Http2Agent() {}
 
-    private static final Logger log = LoggerFactory.getLogger(Http2Agent.class);
-
-    private static boolean hasBeenInstalled = false;
-
     /**
-     * Installs the jetty-alpn-agent dynamically.
-     * <p>
-     * This method protects itself from multiple invocations so may be called in multiple places, but will only
-     * ever invoke the installation once.
+     * No-op.
+     * @deprecated this method no longer has any effect and is safe to stop calling.
      */
+    @Deprecated
     public static synchronized void install() {
-        if (hasBeenInstalled) {
-            return;
-        }
-
-        try {
-            AgentLoader.loadAgentClass(Http2Agent.class.getName(), "");
-            hasBeenInstalled = true;
-        } catch (Exception e) {
-            log.warn("Unable to dynamically install jetty-alpn-agent via ea-agent-loader, proceeding anyway...", e);
-        }
+        // no-op
+        new Exception("Http2Agent.install() no longer does anything, please remove all invocations from your codebase")
```

Comment:
@dansanduleac I think it's much safer to deprecate to no-op than deprecate and begin throwing exceptions/this is not a reasonable fallback behavior.

Reply:
This does not throw

Reply:
I see. given that we're going to produce deprecation warnings I don't think logging an exception here is necessary.

Reply:
Realistically, these warnings are mostly only going to appear in gradle test output (as they're not structured JSON logs). Given that the gradle compilation output should already contain a deprecation warning I don't really want to start a precedent of adding exceptions every time we deprecate something.